src/types.ts

```diff
@@ -0,0 +1,78 @@
+import { promises as fs } from 'fs'
+import { join } from 'path'
+
+import fetch from 'node-fetch'
+
+import type { DenoBridge } from './bridge.js'
+
+const TYPES_URL = 'https://edge.netlify.com'
+
+// eslint-disable-next-line complexity, max-statements
```

Comment:
should we disable complexity and max-statements globally? this is the sort of thing that code reviews are better at catching

Reply:
Good call! Done in https://github.com/netlify/edge-bundler/pull/66/commits/bb8aaf54251f32317f0e5de19f69bbad45010bad. We should look into disabling that at in https://github.com/netlify/eslint-config-node/, really.